### DISCUSSION TITLE
Does Open Weather Map integration work?

### DISCUSSION THREAD

**Discussion by aeroperf** [JanDeDobbeleer/oh-my-posh#1249] [Troubleshoot] [answered]
I've been trying to get Open Weather Map integration to work, but have not been successful. I've verified that my API key works via the OWM website documentation. But I cannot get any weather information to show in the prompt. Can someone verify that it works? Thanks!

**Comment by lnu** [comment, marked as answer]
it works on my side, at least. Can you check the output of `write-poshdebug`? you should see the request made to owm(take care of not copy pasting your api key here). Please note the default timeout for http calls is really short(20ms), so you probably have to increase it(`http_timeout`). The response is also cached for 10 minutes by default.

My config:
```
        {
          "powerline_symbol": "\ue0b6",
          "foreground": "#ffffff",
          "background": "#389c09",
          "invert_powerline": true,
          "properties": {
            "apikey": "YOURAPIKEY",
            "location": "BASTOGNE,BE",
            "units": "metric",
            "enable_hyperlink": true,
            "http_timeout": 400,
            "cache_timeout": 15,
            "template": "[{{.Weather}} ({{.Temperature}}{{.UnitIcon}})]({{.URL}})"
          },
          "type": "owm",
          "style": "powerline"
        }
```

**Reply by aeroperf** [reply]
Thank you for your answer. Turns out it was the timeout. I has it up to 250 ms, which was not enough, but your 400 did the trick.
